lib/metric-config-parser/metric_config_parser/data_source.py

```diff
@@ -190,6 +206,7 @@ def resolve(
             "friendly_name",
             "description",
             "columns_as_dimensions",
+            "aggregation_units",
```

Comment:
I _think_ this guards against my comment above, but is there a way we can be sure? If someone writes a config and omits the parameter, then here nothing will be set in the `params` dict, so nothing will be passed to `DataSource.__init__`, so it should use the `[AnalysisUnit.CLIENT]` default then... I think...